app/src/main/res/values/strings.xml

```diff
@@ -335,4 +335,5 @@
     <string name="create_edit_post_close">Close</string>
     <string name="lang_language">Language</string>
     <string name="blur_nsfw">Blur NSFW images</string>
+    <string name="post_listing_view_source">View Post Source</string>
```

Comment:
Agree with @twizmwazin , either re-use the `comment_node_view_source` string, or just make this one `View Source`

Reply:
I think he meant can't I reuse the `viewSource` variable to change the button text, but @twizmwazin passing in that variable, all the way from PostListing, just to change the button text feels bad

Reply:
@dessalines Isn't it better that it has it's own resource string? Somewhere along the line someone might change `comment_node_view_source` not thinking that it will also change for the Post Listing...